app/views/admin/settings/forms/_smtp.html.erb

```diff
@@ -1,22 +1,46 @@
+<%= javascript_packs_with_chunks_tag "admin/config/smtp", defer: true %>
+<% only_sendgrid_enabled = ForemInstance.sendgrid_enabled? && !Settings::SMTP.provided_minimum_settings? %>
```

Comment:
Could it make sense to move this whole check into `ForemInstance.only_sendgrid_enabled?` I'm just not super comfortable with views knowing **anything** about business logic.

Reply:
Yip will do - this was still a Draft, was just getting it working and then planning to polish up before review.

Reply:
Yes, sorry, I was a bit too early here 😊